test/built-ins/Number/prototype/toFixed/return-values.js

```diff
@@ -0,0 +1,12 @@
+// Copyright (C) 2016 The V8 Project authors. All rights reserved.
```

Comment:
@Yaffle this change is necessary if you're not on the V8 team, which I don't really know.

I need confirm but if you don't want to worry about it, the best default option would be setting the Copyright to ECMA International. cc @bterlson